swagger.yaml

```diff
@@ -331,23 +331,67 @@ paths:
           type: integer
       responses:
         200:
-          description: 'successful operation'
+          description: 'Successful operation'
           content:
             application/json:
               schema:
                 type: array
                 items:
                   $ref: '#/components/schemas/Device'
         500:
-          description: 'internal server error'
+          description: 'Internal server error'
+    post:
+      description: Adds a new device to MPS
```

Comment:
do you wanna mention the requirement to insert into vault separately?

Reply:
Yes, will add that.